builtin/docker/platform.go

```diff
@@ -220,10 +220,17 @@ func (p *Platform) Status(
 	st.Update("Determining overall container health...")
 	if result.Health == sdk.StatusReport_READY {
 		st.Step(terminal.StatusOK, fmt.Sprintf("Container %q is reporting ready!", containerInfo.Name))
-	} else if result.Health == sdk.StatusReport_PARTIAL {
-		st.Step(terminal.StatusWarn, fmt.Sprintf("Container %q is reporting partially available!", containerInfo.Name))
 	} else {
-		st.Step(terminal.StatusError, fmt.Sprintf("Container %q is reporting not ready!", containerInfo.Name))
+		if result.Health == sdk.StatusReport_PARTIAL {
+			st.Step(terminal.StatusWarn, fmt.Sprintf("Container %q is reporting partially available!", containerInfo.Name))
+		} else {
+			st.Step(terminal.StatusError, fmt.Sprintf("Container %q is reporting not ready!", containerInfo.Name))
+		}
+
+		// Extra advisory wording to let user know that the deployment could be still starting up
+		// if the report was generated immediately after it was deployed or released.
+		st.Step(terminal.StatusWarn, "Waypoint detected that the current deployment "+
+			"is not ready, however your\napplication might be available or still starting up.")
```

Comment:
I think in this case I'd probably nix the `\n` and just have it be a long line. I don't want plugin authors to get in the habit of trying to control the output by adding newlines to step statuses, etc.

Reply:
Sounds good, maybe I'll adjust it to be similar to the long message we print for kind clusters for the serverinstall:

https://github.com/hashicorp/waypoint/blob/e26abbc526af9ad049fb89206d8dc6f228154d0e/internal/serverinstall/k8s.go#L1471-L1478